src/components/CourseCard/EmailSettingsModal.jsx

```diff
@@ -68,64 +69,63 @@ class EmailSettingsModal extends Component {
     const { title } = this.props;
 
     return (
-      <>
-        <Modal
-          title={`Email Settings for ${title}`}
-          body={
-            <>
-              {error && (
-                <StatusAlert
-                  alertType="danger"
-                  dialog={
-                    <div className="d-flex">
-                      <div>
-                        <FontAwesomeIcon className="mr-3" icon={faExclamationTriangle} />
-                      </div>
-                      <div>
-                        An error occurred while saving your email settings. Please try again.
-                      </div>
+      <Modal
```

Comment:
This diff is because I'm removing the unnecessary `<>` that was wrapping the `Modal` component.